Src/GoogleApis.Auth/OAuth2/ITokenAccess.cs

```diff
@@ -22,19 +22,24 @@
 namespace Google.Apis.Auth.OAuth2
 {
     /// <summary>
```

Comment:
Ok, ITokenAccess I already moved to the PCL project.  I also moved JsonCredentialParameter, but I needed to make it public to be able to do that (it was internal before). I am happy to move JsonCredentialParameter back if you agree (I don't see a reason why we should expose this class publicly).